serial/serbridge.c

```diff
@@ -469,6 +469,8 @@ serbridgeInitPins()
 }
 
 // Start transparent serial bridge TCP server on specified port (typ. 23)
+// Here is where we need to change the ports. But how do we do this from the Ajax call?
+
```

Comment:
Delete left-over comment? You have that here https://github.com/jeelabs/esp-link/pull/205/files#diff-b6cc56038011d1226a2ee85cb280b897R47